src/bo4e/enum/messgroesse.py

```diff
@@ -0,0 +1,22 @@
+# pylint: disable=missing-module-docstring
+from bo4e.enum.strenum import StrEnum
+
+
+class Messgroesse(StrEnum):
+    """
+    Gibt die physikalische Größe an, die gemessen wurde.
+    """
+
+    STROM = "STROM"  #: STROM
+    SPANNUNG = "SPANNUNG"  #: SPANNUNG
+    WIRKLEISTUNG = "WIRKLEISTUNG"  #: WIRKLEISTUNG
+    BLINDLEISTUNG = "BLINDLEISTUNG"  #: BLINDLEISTUNG
+    DRUCK = "DRUCK"  #: DRUCK
+    LASTGANG = "LASTGANG"  #: LASTGANG
+    LASTPROFIL = "LASTPROFIL"  #: LASTPROFIL
+    TEMPERATUR = "TEMPERATUR"  #: TEMPERATUR
+    ZZAHL = "ZZAHL"  #: ZZAHL
```

Comment:
should it be really `ZZAHL` or is this a typo in the BO4E docu?

Reply:
ja, das ist schlecht dokumentiert. gemeint ist die Zustandszahl, wenigstens in der doku hätte man das ausschreiben können:
https://github.com/Hochfrequenz/BO4E-python/commit/9502da033143c12f1e5a33d6f9912ad80d7bcbb7#diff-6a88a2daf8ab39c84a4d548546ab3e477b5f724a1e41cca39acebc63878668e0

Reply:
All other comments are in ALLCAP, so should be ZUSTANDSZAHL ;)

Reply:
https://github.com/Hochfrequenz/BO4E-python/pull/218/commits/33ecadc5233086b776dafdd5197091c045dd3993

jetzt alles title case. das war nur all caps, weil es autogeneriert war und es überhaupt keine offizielle doku dazu gibt: https://www.bo4e.de/dokumentation/enumerations/enum-messgroesse

Reply:
I see

Reply:
wenn man das genau ansieht kommt mir auch das grauen wegen der vermischung von singular und plural.